src/cnc/flavors/aws/ecs/1/build/build_functions.sh.j2

```diff
@@ -5,7 +5,7 @@ if ! command -v bash >/dev/null 2>&1; then
     exit 1
 fi
 
-
+{% if service.is_backend %}
```

Comment:
This breaks the frontend build now. There is no harm to having functions defined, I think we should remove this condition.